packages/a11y/package.json

```diff
@@ -6,7 +6,8 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
-		"a11y",
+		"gutenberg",
+		"accessibility",
```

Comment:
```suggestion
		"accessibility",
		"a11y",
```

Reply:
Can we leave `a11y`? It seems a [common label](https://www.npmjs.com/search?q=a11y).